.tekton/pull-request.yaml

```diff
@@ -17,7 +17,7 @@ metadata:
       ) && (
           // Don't run pipeline if only these files changed. Note that 'files.all' is an object
           // coming from PaC (list of all changed files) while the second 'all' is a CEL macro.
-          !files.all.all(x, x.matches(r"^(renovate\.json|.*\.md|\.github/.*)$"))
+          !files.all.all(x, x.matches(r"^(renovate\.json|.*\.md|\.github/.*|\.tekton/.*)$"))
```

Comment:
Why do we want to exclude .tekton?

I'd like to see if changes in tekton are valid and will pass testing before merging them

Reply:
Yeah I'm just testing if the exclusion works (and found out that `Red Hat Konflux / build-definitions-pull-request` somehow became a required check, so we'll need to disable that)

Reply:
\o/ thanks @arewm for removing the required check, now PRs wil no longer be blocked in cases where the .tekton/pull-request pipeline doesn't need to run

Reply:
Dropped the `temp` commit, ready for review